bin/commands/transform.js

```diff
@@ -16,10 +16,12 @@ const fuzzyPathOptions = {
   excludePath: (nodePath) =>
     nodePath.includes('node_modules') ||
     nodePath.includes('build') ||
+    // Exclude all dot files
```

Comment:
Could you add a comment to explain the difference between the two regex lines? I see one is .includes and the other .match, but it's not obvious which does what

Reply:
I believe they are both using `.match`.  Both these blocks do the same thing the `excludePath` is for the initial list and the `excludeFilter` is for the fuzzy short list

```javascript
const fuzzyPathOptions = {
  type: 'fuzzypath',
  excludePath: (nodePath) =>
    nodePath.includes('node_modules') ||
    nodePath.includes('build') ||
    // Exclude all dot files
    nodePath.match(/^\/?(?:\w+\/)*(\.\w+)/),
  excludeFilter: (nodePath) =>
    nodePath.includes('node_modules') ||
    nodePath.includes('build') ||
    // Exclude all dot files
    nodePath.match(/^\/?(?:\w+\/)*(\.\w+)/),
  suggestOnly: false,
};
```